tools/cjxl_ng_main.cc

```diff
@@ -35,9 +46,9 @@ DEFINE_bool(encoder_version, false,
             "Print encoder library version number and exit.");
 
 DEFINE_bool(add_jpeg_frame, false,
-            // This supersedes --jpeg_transcode
-            "Use JxlEncoderAddJPEGFrame to add a JPEG frame, "
-            "rather than JxlEncoderAddImageFrame.");
+            "Use JxlEncoderAddJPEGFrame to add a JPEG frame "
+            "(i.e. transcoding JPEG), rather than using "
+            "JxlEncoderAddImageFrame.");
```

Comment:
Current cjxl does this by default (including jbrd) when the input is a jpeg. It would be nice if we can keep that behavior.

Reply:
Changed behavior to: if input is JPEG, do lossless transcoding by default, but say on stderr that cjxl_ng is relying on that default, asking the user to make a choice in order to silence that message.

Reply:
SGTM.